mne/viz/tests/test_3d.py

```diff
@@ -427,41 +427,80 @@ def test_snapshot_brain_montage():
 @testing.requires_testing_data
 @requires_nibabel()
 @requires_version('nilearn', '0.4')
-def test_plot_volume_source_estimates():
+@pytest.mark.parametrize('mode, stype, init_t, want_t, init_p, want_p', [
+    ('glass_brain', 's', None, 2, None, (-33.3, 16.0, 63.7)),
+    ('stat_map', 'vec', 1, 1, None, (15.7, 16.0, -6.3)),
+    ('glass_brain', 'vec', None, 2, (10, -10, 20), (8.7, -12.0, 21.7)),
+    ('stat_map', 's', 1, 1, (-10, 5, 10), (-12.3, 2.0, 7.7))])
```

Comment:
this does not slow down the tests with so many options?

Reply:
It does

Reply:
can we factor something out with a fixture + scope?

Reply:
no not easily because `nilearn.plotting` is the slow part, not our code